### PR TITLE
Changed the borderColor of milestone items 

### PR DESCRIPTION
**Pull request by rustybatmobile**
…t as asked

Update: 
## Changes proposed

The border color of the milestone items have been changed to lightgrey as requested. 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7159"></a>

### CODE REVIEW

File: components/user/UserMilestone.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function UserMilestone({ milestone, isGoal }) {
     <li
       className={`py-4 ${isGoal ? "opacity-50" : ""}`}
       style={{
-        borderColor: milestone.color,
+        borderColor: "lightgrey",
```
Comment (eddiejaoude) [inline components/user/UserMilestone.js:13]: This can be achieved with a tailwind css class and the style is no longe required

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution 👍 I left an inline comment

**Comment by SaraJaoude** [comment]
@rustybatmobile please can you update the comment to link to the Issue this would solve

**Comment by SaraJaoude** [comment]
> rustybatmobile please can you update the comment to link to the Issue this would solve

@rustybatmobile I have now updated this - please ensure that in any future contributions you link the Issue that your PR is looking to solve. How are you getting on with the comments made by the reviewers?

**Comment by SaraJaoude** [comment]
I will be closing this PR due to lack of activity.

**Pull request closed without merging**